decode.go

```diff
@@ -602,32 +634,38 @@ func (bttm ByteStringToTimeMode) valid() bool {
 	return bttm >= 0 && bttm < maxByteStringToTimeMode
 }
 
-// ByteSliceExpectedEncodingMode specifies how to decode a byte string NOT enclosed in an "expected
-// later encoding" tag (RFC 8949 Section 3.4.5.2) into a Go byte slice.
-type ByteSliceExpectedEncodingMode int
+// ByteStringExpectedFormatMode specifies how to decode CBOR byte string into Go byte slice
```

Comment:
Makes sense.  The old suffix "EncodingMode" for a decoding mode name would have been confusing.